sway-ir/src/instruction.rs

```diff
@@ -163,6 +165,7 @@ impl Instruction {
             Instruction::GetStorageKey => Some(Type::B256),
             Instruction::InsertElement { array, .. } => array.get_type(context),
             Instruction::InsertValue { aggregate, .. } => aggregate.get_type(context),
+            Instruction::IntToPtr(_, ty) => Some(*ty),
```

Comment:
If I was going to be pedantic I might ask for this to be moved below next to `GetPointer` since they're semantically similar.  But I'm not. ;)

Reply:
Oh I thought I was following alphabetical order here but I just noticed that this list is grouped into sublists :)

Reply:
Moved it to right after `GetPointer` :)